graphene/utils/props.py

```diff
@@ -2,7 +2,7 @@ class _OldClass:
     pass
 
 
-class _NewClass(object):
+class _NewClass:
```

Comment:
This module can be simplified a bit now than Python 2 is not supported - this `_OldClass` and `_NewClass` stuff was purely to support Python 2.